src/ralph/security/migrations/0007_auto_20170331_0934.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
```

Comment:
0007 is custom migrations. i'd like to have them seperated unless there is a reason.
Is there a reason? :)